code/modules/mob/living/simple_animal/hostile/reenactors.dm

```diff
@@ -0,0 +1,103 @@
+/*
+-!WARNING!-
+-POTENTIAL TRIGGER-INDUCING ITEMS AHEAD-
+-TREAD WITH CAUTION-
+*/
+
+/mob/living/simple_animal/hostile/wwii
+	name = "World War II Reenactor"
```

Comment:
cheridan himself said in the original away mission PR he didn't like the idea of adding nazis so this was the compromise.

convince him ss13 needs real nazis and i'll change them back